src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -98,6 +103,12 @@ public Command parseCommand(String userInput) throws ParseException {
         case DoneCommand.COMMAND_WORD:
             return new DoneCommandParser().parse(description, flags);
 
+        case JokeCommand.COMMAND_WORD:
+            return new JokeCommand(jokeList);
```

Comment:
Is there a reason why the jokeList should be stored in the parser instead of like the ItemModel? I was thinking that the jokeList should be stored within the model since the execution of the command already require the model.